examples/pytorch/logging/pytorch_log_model.ipynb

```diff
@@ -0,0 +1,460 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "7066087c-6947-4a50-a7fb-6ab71c004ccf",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "# Log a pytorch model\n",
+    "\n",
+    "In this guide we will:\n",
```

Comment:
Let's also add some introduction text at the beginning. For example:

This guide will walk you through how to save your PyTorch model to MLflow and load the saved model for inference. Saving a pretrained/finetuned model in MLflow allows you to easily share the model or deploy it to production.